importers/djornl/parser.py

```diff
@@ -284,13 +331,42 @@ def process_file(self, file, remap_fn, store_fn, err_list, validator=None):
         if not n_stored:
             err_list.append(f"{file['path']}: no valid data found")
 
+    def store_parsed_edge_data(self, datum):
```

Comment:
This was defined under the `load_edges` method before - moving it and the other similar methods (`store_parsed_node_data`) out to be top level methods.